docs/adr/adr_003_bundling_collection_data.md

```diff
@@ -1,7 +1,7 @@
 # Bundling Collection Data for Download
 
 
-* Status: building
+* Status: landed, target version 130
```

Comment:
You called the curse 😜 

I think having it as `accepted` or `planned` is enough

Reply:
10-4.